base/reduce.jl

```diff
@@ -396,7 +396,11 @@ end
 
 
 ## prod
+"""
+    prod(f, iter)
```

Comment:
not sure. try and see what the html looks like locally? guess the exact syntax in the doc index files doesn't matter too much as long as the html comes out right